### PR TITLE
Update centre-de-recherche-sur-les-civilisations-de-l-asie-orientale.csl

### PR DESCRIPTION
**Pull request by PhilPons**
Removal of small caps

### CODE REVIEW

File: centre-de-recherche-sur-les-civilisations-de-l-asie-orientale.csl [hunk 1]
```diff
@@ -218,21 +218,19 @@
...
-              <if variable="locator" match="any">
-                <text variable="locator" prefix="p. "/>
-              </if>
-              <else-if variable="locator" match="none">
-                <label variable="page" form="short"/>
-                <text variable="page"/>
-              </else-if>
-            </choose>
-          </group>
+        <date variable="issued">
```
Comment (bwiernik) [inline centre-de-recherche-sur-les-civilisations-de-l-asie-orientale.csl:221]: Please revert this change. This will create erroneous commas after the year if the item has no locator or page.

File: centre-de-recherche-sur-les-civilisations-de-l-asie-orientale.csl [hunk 2]
```diff
@@ -218,21 +218,19 @@
...
+          <date-part name="year" suffix=", "/>
+        </date>
+        <group delimiter=" ">
+          <choose>
+            <if variable="locator" match="any">
+              <text variable="locator" prefix="p. "/>
```
Comment (bwiernik) [inline centre-de-recherche-sur-les-civilisations-de-l-asie-orientale.csl:227]:
```suggestion
              <label variable="locator" form="short"/>
              <text variable="locator"/>
```

File: centre-de-recherche-sur-les-civilisations-de-l-asie-orientale.csl [hunk 3]
```diff
@@ -361,7 +359,14 @@
   <macro name="issue">
     <choose>
       <if is-numeric="issue">
-        <text variable="issue" form="short" prefix=", n˚ "/>
+        <choose>
+          <if variable="volume">
+            <text variable="issue" form="short" prefix=", n˚ "/>
```
Comment (bwiernik) [inline centre-de-recherche-sur-les-civilisations-de-l-asie-orientale.csl:364]: Do not include a preceding comma or space in a prefix like this. Please move the comma to a delimiter on a group element.

**Review by bwiernik — CHANGES_REQUESTED** [review]
Thanks! A few comments to address.

**Comment by bwiernik** [comment]
@PhilPons Can you take care of the issues above?

File: centre-de-recherche-sur-les-civilisations-de-l-asie-orientale.csl (hunk 1 above)
Comment (PhilPons) [reply centre-de-recherche-sur-les-civilisations-de-l-asie-orientale.csl:221]: OK

File: centre-de-recherche-sur-les-civilisations-de-l-asie-orientale.csl (hunk 2 above)
Comment (PhilPons) [reply centre-de-recherche-sur-les-civilisations-de-l-asie-orientale.csl:227]: I agree with that too

File: centre-de-recherche-sur-les-civilisations-de-l-asie-orientale.csl (hunk 3 above)
Comment (PhilPons) [reply centre-de-recherche-sur-les-civilisations-de-l-asie-orientale.csl:364]: Very well

**Review by PhilPons — COMMENTED** [review]
I agree with everything. Thanks for your help!

**Comment by adam3smith** [comment]
OK, merging this, though the style would definitely benefit from someone going through it carefully for robustness
